client/my-sites/plan-features-2023-grid/style.scss

```diff
@@ -875,6 +885,9 @@ body.is-section-signup.is-white-signup .is-onboarding-2023-pricing-grid .signup_
 }
 
 .plan-comparison-grid {
+	.plan-features-2023-gridrison__actions-buttons {
```

Comment:
It's how I found it :D.
In the spirit of avoiding outside (of Plans comparison) changes as much as possible, I left it alone.
Considering there's more parallel work happening, I'd like to keep it as is to avoid unnecessary conflicts.